src/pages/achievement/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,108 @@
+import { IconNames } from '@blueprintjs/icons';
+import React, { useEffect, useState } from 'react';
+
+import Constants from '../../../commons/utils/Constants';
+import { abilityColor } from '../../../features/achievement/AchievementConstants';
+import { FilterStatus } from '../../../features/achievement/AchievementTypes';
+import AchievementFilter from './subcomponents/AchievementFilter';
+import AchievementOverview from './subcomponents/AchievementOverview';
+import AchievementTask from './subcomponents/AchievementTask';
+import AchievementView from './subcomponents/AchievementView';
+import AchievementInferencer from './subcomponents/utils/AchievementInferencer';
+
+export type DispatchProps = {
+  handleAchievementsFetch: () => void;
+};
+
+export type StateProps = {
+  inferencer: AchievementInferencer;
+  name?: string;
+  group: string | null;
+};
+
+function Dashboard(props: DispatchProps & StateProps) {
+  const { inferencer, name, group, handleAchievementsFetch } = props;
+
+  useEffect(() => {
+    if (Constants.useBackend) {
+      handleAchievementsFetch();
+    }
+  }, [handleAchievementsFetch]);
+
+  const [filterStatus, setFilterStatus] = useState<FilterStatus>(FilterStatus.ALL);
+  const [viewId, setViewId] = useState<number>(-1);
+
+  const handleFilterColor = (status: FilterStatus) => {
+    return status === filterStatus ? '#2dd1f9' : '#ffffff';
```

Comment:
It'll be nice if you can give the name to these colors for easy reference. 

const Colors = {
red = "ff0000"
orange = ...

}

and then use Colors.navyBlue